deploy/charts/cert-manager/values.yaml

```diff
@@ -341,6 +341,10 @@ webhook:
   # Path to a file containing a WebhookConfiguration object used to configure the webhook
   # - --config=<path-to-config-file>
 
+  # Comma separated list of feature gates that should be enabled on the
+  # webhok pod.
```

Comment:
```suggestion
  # webhook pod.
```
nit

Reply:
fixed in https://github.com/cert-manager/cert-manager/pull/6093/commits/55ebaa31b58fdf92667455beb049819ce8b36376